src/main/scala/org/broadinstitute/dsde/firecloud/service/UserService.scala

```diff
@@ -221,4 +205,11 @@ class UserService(rawlsDAO: RawlsDAO, thurloeDAO: ThurloeDAO, googleServicesDAO:
       }
     }
   }
+
+  def getUserProfileGoogle(userToken: UserInfo): Future[PerRequestMessage] = {
```

Comment:
this call was previously made directly from the route/API layer, which as i noted in other places, i don't like